source/common/config/api_type_oracle.cc

```diff
@@ -31,5 +31,26 @@ ApiTypeOracle::getEarlierVersionMessageTypeName(const std::string& message_type)
   }
   return absl::nullopt;
 }
+
+const absl::optional<std::string> ApiTypeOracle::getEarlierTypeUrl(const std::string& type_url) {
+  char delimeter = '/';
+  size_t type_name_start = type_url.size();
+  for (size_t i = 0; i < type_url.size(); i++) {
```

Comment:
Why not use string::find here?

Reply:
Yes, that makes sense. I will use string::find.